app/views/users/show.html.erb

```diff
@@ -1 +1,35 @@
-<h1>Show a specific user from the database</h1>
\ No newline at end of file
+<div class = "user_layout">
+<div class = "user_container">
+<img src="<%= @user.photo %>" alt="user" />
+<div class = "user_content">
+<h2><%= @user.name %></h2>
+<h4>Number of posts: <%= @user.posts.count %></h4>
+</div>
+</div>
+
+<div class = "bio">
+<h2>Bio</h2>
+<p><%= @user.bio %></p>
+</div>
+
+<div class= "recent_posts">
+<%= @user.recent_posts.each do |post| %>
```

Comment:
- [ ] There is an issue with the line `<%= @user.recent_posts.each do |post| %>`. The `each` method returns the original array and not nil, so it will be rendered in the view. To fix this, you can change <%= to <% to prevent the array from being rendered. 👇🏽 

![image](https://github.com/edubew/blog-box/assets/58642949/fb8db9e4-000a-4743-9497-6d96517ac6b4)